numpy/random/_pickle.py

```diff
@@ -14,70 +14,67 @@
                  }
```

Comment:
It would be nice if we could refactor this to avoid the circular import `_pickle` -> _generator/mtrand -> `_pickle`

Reply:
well, not really circular since `_generator`/`mtrand` import `_pickle` in the `__reduce__` method, but the pattern is convoluted.